### DISCUSSION TITLE
Where is all .py files in the Nightly release?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#5941] [Q&A] [answered]
Compared to the package produced with `make linux-package` command, I do not see any directories containing the Python code for both kitty itself and the associated kittens anywhere within the latest Nighty build.

How does it even work?

**Comment by kovidgoyal** [comment, marked as answer]
Magic :) The compiled python bytecode is all loaded from asingle memory
mapped file for maximum performance. See my bypy project if you are
interested in how its done.
